ci/do.sh

```diff
@@ -109,12 +114,19 @@ if [ "x${ONLY_REBUILD}" != "x1" -a "x${ONLY_INSTALL}" != "x1" -a "x${ONLY_TEST}"
     else 
       configure_opt='--prefix=/opt/pbs --enable-ptl'
     fi
+    if [ "x${BUILD_MODE}" == "xkerberos" ]; then
+      configure_opt="${configure_opt} --with-krbauth PATH_KRB5_CONFIG=/usr/bin/krb5-config"
+    fi
```

Comment:
This condition is not useful here as BUILD_MODE variable is not passed in CI.

Reply:
Thank you, The condition is removed. It is not needed.